mkdocs.yml

```diff
@@ -39,3 +39,15 @@ plugins:
             show_source: true
             show_signature_annotations: true
 
+#https://squidfunk.github.io/mkdocs-material/reference/math/
+markdown_extensions:
```

Comment:
Do these have to be installed? We might need to add them to the requirements.txt file before merging

Reply:
will add it.